cirq-google/cirq_google/engine/engine_job.py

```diff
@@ -299,12 +299,15 @@ async def _await_result_async(self) -> quantum.QuantumResult:
                     'Internal error: The job response type is not recognized.'
                 )  # pragma: no cover
 
+        # The desired rate limit is 1000 QPM which is ~15 QPS
```

Comment:
Can you explain how this results in 15QPS? AFAICT, there's no time-based component here to rate-limit the requests.

Reply:
IIUC, it's 15QPS if you quite generously assume jobs complete in 1s. I'm happy to change the limiter capacity to reflect a more accurate job completion rate #